site/zh/blog/new-in-devtools-106/index.md

```diff
@@ -22,32 +22,46 @@ draft: true
 {% YouTube id='5gBqTXctxO8' %}
 
 <!-- start: translation instructions -->
+开始：翻译使用说明
```

Comment:
```suggestion
```

`<!-- start: translation instructions -->` - These are instructions for you. Don't need to translate these